framework/tlvf/yaml/tlvf/wfa_map/tlvApMetric.yaml

```diff
@@ -9,7 +9,7 @@ tlvApMetric:
     _type: eTlvTypeMap
     _value_const: TLV_AP_METRIC 
   length: uint16_t
-  bssid: sAddress
+  bssid: sMacAddr
```

Comment:
I could understand why it is right to replace sAddress to sMacAddr, but I don't understand how it is relevant to this PR, since I don't see you use it. So this change should be in a separate PR.

Reply:
Its a small bug related to the PR.
I'm not sure that opening a new PR for name change is a good idea.
but if you already spent the time to comment on that i'll fix it in separate commit .